docs/proposal/bypass.md

```diff
@@ -0,0 +1,88 @@
+
+## Bypass Function Design
+
+Bypass is a general mesh function used to remove sidecar/kmesh mesh services from the traffic path. The following two scenes show the traffic path after the bypass function is enabled.
+
+### Scene 1: sidecar
+
+Among these two figures, the mesh service is enabled in the cluster to take over the traffic. The first figure shows the traffic path before the bypass function is enabled. We can see from it that the traffic path is server A -> sidecar -> sidecar - > server B, when the communication between server A and server B fails, from the black box level, the user cannot determine whether there is a problem with the sidecar or the server itself. In Figure 2, the bypass function of pod1 is enabled, so that the sidecar in pod1 is removed from the traffic path. If server A and server B can communicate normally at this time, it proves that there is a problem with the sidecar.
+
+![alt text](pics/sidecar_pre_bypass.svg)
+
+![alt text](pics/sidecar_bypass.svg)
+
+### Scene 2: kmesh
+
+In Figure 1, the traffic path is service A->kmesh->service B. In Figure 2, after the bypass function is enabled, it will be judged in the eBPF program of kmesh that the bypass function has been enabled, so kmesh will not take over the data sent by service A, and the traffic path is service A->service B. If service A and service B can communicate normally at this time, it proves that there is a problem with kmesh.
+
+![alt text](pics/kmesh_pre_bypass.svg)
+
+![alt text](pics/kmesh_bypass.svg)
+
+## Bypass fields
+
+```c
+struct bpf_map_def SEC("maps") bypass_fliter_map = {
+    .type = BPF_MAP_TYPE_HASH,
+    .key_size = sizeof(__u32),
+    .value_size = sizeof(__u8),
+    .max_entries = 256,
+};
+```
+
+The key represents the IP address of the pod, and the value represents whether the bypass function is enabled for the current pod, represented by 0/1. When this ebpf program is executed, it will check whether the corresponding pod has the bypass function enabled. If enabled, all subsequent iptables rules will be short-circuited and inbound and outbound traffic will be short-circuited, otherwise, subsequent iptables rules will not be short-circuited.
+
+## How to enable bypass
+
+Users can label pods with the following command:
+
+```shell
+kubectl label pod <pod_name> bypass=enabled
+```
+
+This causes the inbound and outbound traffic of the mesh service on the pod to be short-circuited.
+
+## How to watch label
+
+The Kmesh Daemon observes changes in the Kubernetes API Server and implements observation of specific labels using a Field Selector. It communicates with the kube-apiserver, filters the desired resource objects, and thus only observes changes specific to certain labels.
+
+```go
+labelSelector := "bypass"
+
+pod, err := client.CoreV1().Pods(pod.Namespace).Get(context.TODO(), metav1.GetOptions{
+    LabelSelector: labelSelector,  
```

Comment:
labelSelector is not right
add field selector too

Reply:
> labelSelector is not right add field selector too

I've updated